quartz/components/scripts/search.inline.ts

```diff
@@ -222,16 +222,18 @@ document.addEventListener("nav", async (e: unknown) => {
 
   const resultToHTML = ({ slug, title, content, tags }: Item) => {
     const htmlTags = tags.length > 0 ? `<ul>${tags.join("")}</ul>` : ``
-    const button = document.createElement("button")
-    button.classList.add("result-card")
-    button.id = slug
-    button.innerHTML = `<h3>${title}</h3>${htmlTags}<p>${content}</p>`
-    button.addEventListener("click", () => {
+    const itemTile = document.createElement("a")
+    itemTile.classList.add("result-card")
+    itemTile.id = slug
+    itemTile.href = new URL(resolveRelative(currentSlug, slug), location.toString()).toString()
+    itemTile.innerHTML = `<h3>${title}</h3>${htmlTags}<p>${content}</p>`
+    itemTile.addEventListener("click", (event) => {
+      if (event.altKey || event.ctrlKey || event.metaKey || event.shiftKey) return
       const targ = resolveRelative(currentSlug, slug)
       window.spaNavigate(new URL(targ, window.location.toString()))
```

Comment:
i wonder if, now that we have it as a regular `a` tag, we can remove these two lines

could you try and see if that still works and SPA navs?

(haha sorry for all the follow ups, I'm just noticing these as I review)

Reply:
sg no worries happy to try it out.

Reply:
qq: What do you mean by `SPA nav` here? Does it still work when users click on the item?

If that is the case then yes.

Reply:
yeah we specifically hijack navigation to fetch the content diff and replace rather than actually letting the browser do it

https://quartz.jzhao.xyz/features/SPA-Routing

Reply:
If that is the case then removing those two lines still works with SPA nav 😃

Reply:
alright let's remove and :shipit: